miso-web/src/main/java/uk/ac/bbsrc/tgac/miso/webapp/controller/rest/LibraryRestController.java

```diff
@@ -179,4 +179,11 @@ public LibraryDto updateLibrary(@PathVariable("id") Long id, @RequestBody Librar
     return jQueryBackend.get(request, response, uriBuilder, PaginationFilter.project(id));
   }
 
+  @RequestMapping(value = "/query", method = RequestMethod.POST, produces = { "application/json" })
+  @ResponseBody
+  public List<LibraryDto> getSamplesInBulk(@RequestBody List<String> names, HttpServletRequest request, HttpServletResponse response,
+      UriComponentsBuilder uriBuilder) throws IOException {
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove the declaration of thrown exception 'java.io.IOException', as it cannot be thrown from method's body. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3ARedundantThrowsDeclarationCheck)